publisher/product/ontology/build.sh

```diff
@@ -677,41 +691,41 @@ __HERE__
 
 
   log "Getting metadata"
-  "${JENA_ARQ}" $(find "${source_family_root}" -name "MetadataFIBO.rdf" | grep -v "/etc/" | sed "s/^/--data=/") \
+  "${JENA_ARQ}" $(find "${source_family_root}" -name "Metadata${ONTPUB_FAMILY}.rdf" | grep -v "/etc/" | sed "s/^/--data=/") \
```

Comment:
Should be uppercase:
`"${JENA_ARQ}" $(find "${source_family_root}" -name "Metadata${ONTPUB_FAMILY^^}.rdf" | grep -v "/etc/" | sed "s/^/--data=/") \`